packages/api/src/router/post.ts

```diff
@@ -6,4 +8,42 @@ export const postRouter = createTRPCRouter({
       orderBy: { createdAt: "desc" },
     });
   }),
+  createPost: protectedProcedure
+    .input(z.object({ title: z.string(), contents: z.string() }))
+    .mutation(async ({ ctx, input }) => {
+      await ctx.db.post.create({
+        data: {
+          title: input.title,
+          contents: input.contents,
+        },
+      });
+    }),
+  updatePostByID: protectedProcedure
+    .input(
+      z.object({
+        id: z.number(),
```

Comment:
can you specify that the number needs to be >= 0

Reply:
Ok!